app/views/schools/dashboard/show.html.erb

```diff
@@ -1 +1,35 @@
-Dashboard goes here
+<span class="govuk-caption-l"><%= @school.name %></span>
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">Manage your training</h1>
+
+<p class="govuk-body govuk-!-margin-bottom-5">Check and add details for every academic year/cohort.</p>
+
+<h3 class="govuk-heading-m govuk-!-margin-bottom-1">Your cohorts</h3>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header" scope="col">Year</th>
+      <th class="govuk-table__header" scope="col">Programme</th>
+      <th class="govuk-table__header" scope="col">Status</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% @cohorts.each do |cohort| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">
+          <%= govuk_link_to cohort[:cohort].start_year, schools_cohort_path(cohort[:cohort].start_year) %>
```

Comment:
Are the `@cohorts` here actually `@school_cohorts`? 

Also, no hash notation please